custom-connector-sdk/src/main/java/com/amazonaws/appflow/custom/connector/model/connectorconfiguration/auth/OAuth2Defaults.java

```diff
@@ -75,6 +75,20 @@ public interface OAuth2Defaults {
      */
     List<OAuth2GrantType> oAuth2GrantTypesSupported();
 
+    /**
+     * OAuth2 Content type to use in the token request header.
+     */
```

Comment:
Add "@Value.Default" om attributes

Reply:
Ok. Whats the purpose just curious

Reply:
It will take default value when from server this field is not sent back in response.